src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDriver.java

```diff
@@ -524,8 +530,7 @@ enum SQLServerDriverIntProperty {
     STATEMENT_POOLING_CACHE_SIZE("statementPoolingCacheSize", SQLServerConnection.DEFAULT_STATEMENT_POOLING_CACHE_SIZE),
     CANCEL_QUERY_TIMEOUT("cancelQueryTimeout", -1),
     CONNECT_RETRY_COUNT("connectRetryCount", 1, 0, 255),
-    CONNECT_RETRY_INTERVAL("connectRetryInterval", 10, 1, 60),
-    MSI_TOKEN_CACHE_TTL("msiTokenCacheTtl", 3600, 0, Integer.MAX_VALUE);
```

Comment:
If we remove all the connection string-related code for msiTokenCacheTtl, will the driver complain about an unknown option? Or does it ignore unknown options? @lilgreenbird Do you know off the top of your head?

(If it doesn't ignore them, we would not want it to error until we actually remove the deprecated setting.)